src/components/layout/tree/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { List, ListItem, ListItemIcon, ListItemText, Icon, IconButton } from '@material-ui/core';
+import IDoctor from '../../../interfaces/doctor';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        padding: 0,
+        alignItems: 'center',
+        height: "auto"
+    },
+    nested: {
+        paddingLeft: 20,
+        background: "rgba(254, 97, 124, 0.8)",
+        color: "white"
+    },
+    red: {
+        color: "#FE617C"
+    }
+}));
+
+interface IProps {
+    filteredItems: Array<IDoctor>,
+    openItems: any,
+    selectedItems: Array<IDoctor>,
+    handleSetSelectedItem: any,
+    handleOpenItems: any
+}
+
+export default function Tree({ filteredItems, openItems, selectedItems, handleSetSelectedItem, handleOpenItems }: IProps): JSX.Element {
+    const classes = useStyles();
+    return (
+        filteredItems.length > 0
+            ? (
+                <List
+                    component="nav"
+                    aria-labelledby="nested-list-subheader"
+                    className={classes.root}
+                >
```

Comment:
To avoid nesting the whole content, we can return early
```suggestion
    if (filteredItems.length <= 0) {
        return null;
    }

    return (
        <List
               component="nav"
               aria-labelledby="nested-list-subheader"
               className={classes.root}
        >
```